extensions/sql-database-projects/src/test/projectController.test.ts

```diff
@@ -54,23 +54,6 @@ describe('ProjectsController', function (): void {
 
 	describe('project controller operations', function (): void {
 		describe('Project file operations and prompting', function (): void {
-			it('Should create new sqlproj file with correct values', async function (): Promise<void> {
```

Comment:
I removed this test because there isn't a way to pass in the projectGuid to the new project api, so the compare was failing because the guids would be different. There are also already tests for this functionality in [STS](https://github.com/microsoft/sqltoolsservice/blob/main/test/Microsoft.SqlTools.ServiceLayer.IntegrationTests/SqlProjects/SqlProjectsServiceTests.cs#L60-L84) and [DacFx](https://msdata.visualstudio.com/SQLToolsAndLibraries/_git/DacFx?path=/Test/UnitTests/UTProjects/ProjectTests.cs&version=GBmain&line=21&lineEnd=30&lineStartColumn=9&lineEndColumn=10&lineStyle=plain&_a=contents).